### PR TITLE
refresh-chatter on sign-in / bump web-sdk / bug fix

### PR DESCRIPTION
**Pull request by zo-el**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,15 +6,15 @@
     "serve:dev-agent-1": "VUE_APP_WEB_CLIENT_PORT=8888 yarn serve:develop",
     "serve:dev-agent-2": "VUE_APP_WEB_CLIENT_PORT=9999 yarn serve:develop",
     "serve:develop": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holochain' vue-cli-service serve",
-    "serve": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holo-host' VUE_APP_CHAPERONE_SERVER_URL='http://localhost:24273/' vue-cli-service serve",
+    "serve": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holo-host' VUE_APP_CHAPERONE_SERVER_URL='http://localhost:8000/' vue-cli-service serve",
```
Comment (timotree3) [inline package.json:9]: Why is your chaperone at 8000 when mine is at 24273?

File: package.json [hunk 2]
```diff
@@ -62,4 +62,4 @@
       "git add"
     ]
   }
-}
+}
```
Comment (timotree3) [inline package.json:65]:
```suggestion
}

```

**Review by timotree3 — APPROVED** [review]
Looks good

File: package.json (hunk 1 above)
Comment (zo-el) [reply package.json:9]: I am using `make static-server` which just spins a production-level chaperone server on `8000`. I can revert this back.

File: package.json (hunk 2 above)
Comment (zo-el) [reply package.json:65]: why do we need an extra line here?

Comment (timotree3) [reply package.json:65]: typically files end in a newline

Comment (zo-el) [reply package.json:65]: ok

**Pull request merged**